test/functional/shada/shada_spec.lua

```diff
@@ -264,4 +264,21 @@ describe('ShaDa support code', function()
     session:close()
     os.remove('NONE')
   end)
+
+  it('does not store unlisted buffer', function()
+    skip(is_os('win'))
```

Comment:
there's no good reason to skip this on windows. we need to figure out why the test isn't working there

Reply:
Hi Justin,

thanks for looking at the PR again. As mentioned in [this comment](https://github.com/neovim/neovim/pull/21818#issuecomment-1414145819) I've disabled the test, because I couldn't figure out why the test is failing on windows.

I wanted to try, whether the test would succeed on windows after rebasing on the latest master commit - it doesn't.

I am sorry for taking your time and not properly marking this PR as Draft. ✌🏻